cmd/tg_bot.go

```diff
@@ -73,6 +74,8 @@ var tgBotCmd = &cobra.Command{
 			return err
 		}
 
+		monitoring.InitSentry(cfg.Sentry.DSN)
```

Comment:
no, you should init it in root.go
add function like initConfig (just use config.GetConfig and then call init sentry)
and pass that function as argument in cobra.OnInitialize (after initConfig)

Reply:
also add sentry *Hub to Bot structure field (cause it is not required option, we have to determine sentry is on or off)
check sentry.GetHubFromContext() method